src/Zcu/PerThread.zig

```diff
@@ -39,7 +39,6 @@ pub fn astGenFile(
     pt: Zcu.PerThread,
```

Comment:
can you please add doc comments at the top of PerThread.zig? The role of this file is not intuitive to me